server/routes/dev_place/utils.py

```diff
@@ -232,60 +251,138 @@ def chart_config_to_overview_charts(chart_config):
   charts = []
   for page_config_item in chart_config:
     denominator = page_config_item.get("denominator", None)
+    is_map_chart = page_config_item.get("isChoropleth", False)
     chart = Chart(
         category=page_config_item.get("category"),
         description=page_config_item.get("description"),
         scaling=page_config_item.get("scaling"),
         statisticalVariableDcids=page_config_item.get("statsVars", []),
         title=page_config_item.get("title"),
         topicDcids=[],
-        type="MAP" if page_config_item.get("isChoropleth", False) else "LINE",
+        type="MAP" if is_map_chart else "LINE",
         unit=page_config_item.get("unit"),
     )
     if denominator:
       chart.denominator = denominator
+    if is_map_chart:
+      chart.childPlaceType = child_place_type
     charts.append(chart)
 
+    if page_config_item.get("relatedChart", {}).get("title"):
+      page_config_related_chart = page_config_item.get("relatedChart", {})
+      related_chart = Chart(
```

Comment:
I might be missing it but what happens with this related_chart? Is this used somehwere?

Reply:
Good catch. Appended this chart to the `charts` list.